src/pudl/etl/epacems_assets.py

```diff
@@ -148,8 +149,30 @@ def hourly_emissions_epacems(
     partitions = years.map(
         lambda year: process_single_year(
             year,
-            epacamd_eia,
+            clean_epacamd_eia,
             plants_entity_eia,
         )
     )
     return consolidate_partitions(partitions.collect())
+
+
+@asset(
+    ins={
+        "hourly_emissions_epacems": AssetIn(input_manager_key="epacems_io_manager"),
+    }
+)
+def emissions_unit_ids_epacems(
+    hourly_emissions_epacems: dd.DataFrame,
+) -> pd.DataFrame:
+    """Make unique annual plant_id_eia and emissions_unit_id_epa.
+
+    Returns:
+        dataframe with unique set of: "plant_id_eia", "year" and "emissions_unit_id_epa"
+    """
+    epacems_ids = (
+        hourly_emissions_epacems[["plant_id_eia", "year", "emissions_unit_id_epa"]]
+        .drop_duplicates()
+        .compute()
+    )
+
+    return epacems_ids
```

Comment:
even though right now this is only used in the glue_assets, I kept it in cems land bc it feels more cems-y than gluey.

Reply:
Do you feel like this needs to be an asset? Could it just be a line in `def epacamd_eia_subplant_ids`?

Reply:
bc its reading the chonky `hourly_emissions_epacems` even with dask it still takes a little over a minute to make. which is not the end of the world but it definitely makes testing it easier

Reply:
Oh ok, that way, you can rematerialize `epacamd_eia_subplant_ids` without doing the expensive dask computation.